.gitignore

```diff
@@ -0,0 +1 @@
+.DS_Store
```

Comment:
Pls remove this file.

Reply:
I think there is a need for .gitignore file here because when I pulled the code locally, mac added .DS_Store file as a git change.
Better than manually removing .DS_Store from "git add" , it's better to put it in a ignore file.